example/app/components/TaskManager.tsx

```diff
@@ -28,7 +31,10 @@ export const TaskManager: React.FC<{
       // of sync participants to successfully sync everything in the transaction, otherwise
       // no changes propagate and the transaction needs to start over when connectivity allows.
       realm.write(() => {
-        return new Task(realm, description, userId);
+        return realm.create(Task, {
+          description,
+          userId: userId ? userId : 'blah',
```

Comment:
(super minor) Only if we want `'blah'` when `userId` is `undefined`/`null` (not falsy)
```suggestion
          userId: userId ?? 'blah',
```

Reply:
Oh whoops.  I was definitely should make that a nicer string :D